src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -265,18 +287,101 @@ const NetworkModificationNodeEditor = ({ selectedNode }) => {
                     })
-                    .catch((err) => snackError(err.message));
+                    .catch((err) => snackError(err.message))
+                    .finally(() => {
+                        setLaunchLoader(false);
+                    });
             }
         }
-    }, [selectedNode, setModifications, selectedNodeRef, snackError]);
+    }, [
+        selectedNode,
+        setModifications,
+        selectedNodeRef,
+        snackError,
+        dispatch,
+        errorList,
+    ]);
+
+    const fillNotification = useCallback(
+        (study, messageId) => {
+            // (work for all users)
+            const notification = {
+                studyUuid: study.eventData.headers['studyUuid'],
+                nodeUuid: study.eventData.headers['parentNode'],
+            };
+            // specific message id for each action type
+            setMessageId(messageId);
+            dispatch(addNotification(notification));
+        },
+        [dispatch]
+    );
+
+    const manageNotification = useCallback(
+        (study) => {
+            let messageId = '';
+            if (
+                study.eventData.headers['updateType'] === 'creatingInProgress'
+            ) {
+                messageId = 'network_modifications/creatingModification';
+            } else if (
+                study.eventData.headers['updateType'] === 'updatingInProgress'
+            ) {
+                messageId = 'network_modifications/updatingModification';
+            } else if (
+                study.eventData.headers['updateType'] === 'deletingInProgress'
+            ) {
+                messageId = 'network_modifications/deletingModification';
+            }
+            fillNotification(study, messageId);
+        },
+        [fillNotification]
+    );
+
+    useEffect(() => {
+        const updateType = [
+            'creatingInProgress',
+            'updatingInProgress',
+            'deletingInProgress',
+        ];
+        if (studyUpdatedForce.eventData.headers) {
+            if (
+                selectedNodeRef.current.id !==
+                studyUpdatedForce.eventData.headers['parentNode']
+            )
+                return;
+
+            if (
+                updateType.includes(
+                    studyUpdatedForce.eventData.headers['updateType']
+                )
+            ) {
+                manageNotification(studyUpdatedForce);
+            }
+            // notify  finished action (success or error => we remove the loader)
+            // error handling in dialog for each equipment (snackbar with specific error showed only for current user)
+            if (
+                studyUpdatedForce.eventData.headers['updateType'] ===
+                'UPDATE_FINISHED'
+            ) {
+                const notification = {
+                    nodeUuid: studyUpdatedForce.eventData.headers['parentNode'],
+                };
+                dispatch(removeNotificationByNode(notification));
```

Comment:
You could pass the nodeUuid here directly to removeNotificationByNode, why create a notification object ?

Reply:
yes good idea il clean the code to pass directly just the nodeUuid